pages/common/git-reauthor.md

```diff
@@ -0,0 +1,21 @@
+# git reauthor
+
+> Replace details about an author identity, since this command rewrites git's history --force will be needed when `git push` is used.
+> Part of `git-extras`.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-reauthor>.
+
+- Replace the personal email and name of Jack to his work ones:
+
+`git reauthor --old-email jack@perso.me --correct-email jack@work.com --correct-name 'Jack Foobar'`
```

Comment:
Are you sure, you have read our style guide?

```suggestion
`git reauthor --old-email {{old@example.com}} --correct-email {{new@example.com}} --correct-name "{{name}}"`
```

Reply:
Sorry about that, my bad